common/gnome-shell/3.36/sass/_common.scss

```diff
@@ -2654,12 +2654,12 @@ StScrollBar {
   .notification,
   .unlock-dialog-notification-source {
     padding: 12px 6px;
-    border: 1px solid $_bubble_borders_color;
-    background-color: transparentize($osd_bg_color,0.5);
-    color: $_bubble_fg_color;
+    border: 1px solid transparentize(black, 0.9);
+    background-color: transparentize($shell_popup_menu_background_color, 0.5);
+    color: $fg_color;
```

Comment:
My idea was to simply do `@extend %popup_menu;` here for now, same as with `.notification-banner`. While that doesn't follow the default shell theme styling, the 50% transparency looks extremely out-of-place with rest of the Arc-theme design. This can be improved later.

Also I think symbolic icons on the notifications might work better with Arc style. I think you should be able use them here with `> StIcon { -st-icon-style: symbolic; }`.

Reply:
On balance, I agree that the default notification style looks better (although slightly mismatched with buttons on unlock screen)

> Also I think symbolic icons on the notifications might work better with Arc style. I think you should be able use them here with > StIcon { -st-icon-style: symbolic; }.

Good idea, done